queries.sql

```diff
@@ -11,3 +11,56 @@ SELECT * from animals WHERE weight_kg BETWEEN 10.4 and 17.3;
 UPDATE animals
 SET name = 'Agumon'
 WHERE name = 'Agumen';
+
+BEGIN TRANSACTION;
+
+UPDATE animals
+SET species = 'unspecified';
```

Comment:
- [ ] Before doing `rollback`
`SELECT species from animals;` -- verify that change was made

- [ ] After `rollback`
`SELECT species from animals;` -- verify that change was undone